solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -193,7 +194,9 @@ public boolean split(ClusterState clusterState, ZkNodeProps message, NamedList<O
     RTimerTree t;
     if (ccc.getCoreContainer().getNodeConfig().getMetricsConfig().isEnabled()) {
       // check disk space for shard split
-      if (Boolean.parseBoolean(System.getProperty(SHARDSPLIT_CHECKDISKSPACE_ENABLED, "true"))) {
+      if (!message.getBool(
+          SKIP_FREE_SPACE_CHECK,
+          !Boolean.parseBoolean(System.getProperty(SHARDSPLIT_CHECKDISKSPACE_ENABLED, "true")))) {
```

Comment:
Do we need both properties? Doesn't `SHARDSPLIT_CHECKDISKSPACE_ENABLED` handle the case we care about already?

If you are running on HDFS you could set `SHARDSPLIT_CHECKDISKSPACE_ENABLED` to false?

Reply:
It looks like it's already turned off automatically on HDFS: https://github.com/apache/solr/blob/main/solr/modules/hdfs/src/java/org/apache/solr/hdfs/HdfsDirectoryFactory.java#L196

This can be closed then, I guess :)

Reply:
🎉 sorry @gezapeti for the extra work. I was just going through the lucene-solr PRs and noticed this was hanging out there. Didn't realize it was addressed a different way.